dashboard/app/controllers/foorm/simple_survey_forms_controller.rb

```diff
@@ -1,7 +1,31 @@
 module Foorm
-  class SimpleSurveyController < ApplicationController
+  class SimpleSurveyFormsController < ApplicationController
+    # Admin authorization is not used in other Foorm contexts (eg, creating new Form content in levelbuilder).
+    # Admin is the most straightforward way to allow staff to publish surveys, so we use it here.
+    before_action :require_admin, only: [:index, :new, :create]
+
     FOORM_SIMPLE_SURVEY_SUBMIT_API = '/dashboardapi/v1/foorm/simple_survey_submission'
 
+    def create
+      form = Foorm::Form.all.detect {|f| f.key == params[:form_key]}
+      return head :bad_request unless form
+
+      # Admin only page, so return any errors in plain text.
+      begin
+        Foorm::SimpleSurveyForm.create!(
+          kind: params[:kind],
+          path: params[:path],
+          form_name: form.name,
+          form_version: form.version,
+          allow_multiple_submissions: params[:allow_multiple_submissions] == '1'
```

Comment:
Rails checkbox returns '0' when unselected, which is truthy. So, explicitly check for the value returned when checkbox is checked.